features/hq/hq_features/authentication.feature

```diff
@@ -0,0 +1,14 @@
+Feature:
+  As a site admin for osra
+  So that I have access to restricted pages
+  I would have to authenticate first
+
+  Scenario: Restricted pages should redirect unauthenticated users to login page
+    Given I am an unauthenticated user
+    When I go to the "partners" page for the "admin" role
```

Comment:
Since we are testing authentication for the entire application, the partners page is not relevant per se. What do you think about abstracting it as something like `When I try to access a protected page`?

This can be even further abstracted as:
```cucumber
Given I am an unauthenticated visitor
Then I should not be able to access protected areas of the application
```